docs/en/setup/backend/backend-alarm.md

```diff
@@ -22,6 +22,8 @@ Alarm rule is constituted by following keys
 [List of all potential metrics name](#list-of-all-potential-metrics-name).
 - **Include names**. The following entity names are included in this rule. Please follow [Entity name define](#entity-name).
 - **Exclude names**. The following entity names are excluded in this rule. Please follow [Entity name define](#entity-name).
+- **Include names regex**. Provide a regex to include the entity names.
+- **Exclude names regex**. Provide a regex to exclude the exclude names.
```

Comment:
What happens if `include names` and `include names regex` exist both? I think we should indicate in the document.

Reply:
I think both will take effect.